client/src/components/History/SwitchToHistoryLink.vue

```diff
@@ -0,0 +1,69 @@
+<script setup lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faArchive, faBurn } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+import { BLink } from "bootstrap-vue";
+import { computed } from "vue";
+import { useRouter } from "vue-router/composables";
+
+import { HistorySummary } from "@/api";
+import { useHistoryStore } from "@/stores/historyStore";
+
+import LoadingSpan from "@/components/LoadingSpan.vue";
+
+library.add(faArchive, faBurn);
+
+const router = useRouter();
+const historyStore = useHistoryStore();
+
+interface Props {
+    historyId: string;
+}
+
+const props = defineProps<Props>();
+
+const history = computed(() => historyStore.getHistoryById(props.historyId));
+
+const canSwitch = computed(() => !!history.value && !history.value.archived && !history.value.purged);
+
+const actionText = computed(() => (canSwitch.value ? "Switch to" : "View in new tab"));
+
+function onClick(history: HistorySummary) {
+    console.log("onClick", history);
```

Comment:
Is it ok if I drop the console logging ?

Reply:
Sure Thanks! That was a leftover from debugging